src/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "users-geolocation",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "npm run gen-types && tsc -p tsconfig.json",
+    "test": "jest",
+    "gen-types": "graphql-codegen --config codegen.yml",
+    "pack": "npm run build && trace-pkg -c pkg-config.yml"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@mapbox/mapbox-sdk": "^0.12.1",
+    "apollo-server-lambda": "^2.23.0",
+    "apollo-server-plugin-base": "^0.12.0",
+    "bufferutil": "^4.0.3",
+    "encoding": "^0.1.13",
+    "graphql": "^15.5.0",
+    "moment": "^2.29.1",
```

Comment:
Do you think this library is important in this project? If you check the generated bundle, it increases the lambda package drastically. Considering this, would you recommend another alternative?

Reply:
You're completely right. Since my packages were not getting that big, I completely overlooked this. For the basic functionality  I am using of Moment, it is eating a good chunk of my packages (I calculated it to be around 10%). I definitely should go for a smaller package.